sdk/core/azure-core/src/cryptography/md5.cpp

```diff
@@ -194,7 +194,6 @@ class Md5OpenSSL final : public Azure::Core::Cryptography::Hash {
     }
     if (1 != EVP_DigestInit_ex(m_context, EVP_md5(), NULL))
     {
-      EVP_MD_CTX_free(m_context);
```

Comment:
why not?   Seems like it is preventing a leak after `EVP_MD_CTX_new()`

Reply:
m_context is a smart pointer, it is freed in the destructor of m_context. This call does nothing.